common/scala/src/main/resources/application.conf

```diff
@@ -70,9 +70,10 @@ whisk {
                 retention-ms    = 3600000
             }
             invoker {
-                segment-bytes   =  536870912
-                retention-bytes = 1073741824
-                retention-ms    = 172800000
+                segment-bytes     =  536870912
+                retention-bytes   = 1073741824
+                retention-ms      =  172800000
+                max-message-bytes = ${whisk.activation.payload.max}
```

Comment:
Why is this needed?

Reply:
Wanted to seperate kafka and general config, so the value is carried over by configuration not by hardcoding.

I also changed the topic's configurations to be generalized as in: You can provide all possible Kafka keys here and they are transformed to the respective Kafka string.